python/taichi/lang/ops.py

```diff
@@ -546,3 +547,24 @@ def deactivate(l, indices):
 
 def length(l, indices):
     return Expr(_ti_core.insert_len(l.snode.ptr, make_expr_group(indices)))
+
+
+def rescale_index(a, b, I):
```

Comment:
nit: Could you add a docstring like https://github.com/taichi-dev/taichi/blob/abd063c92fa24616118c9feddcc13efc4b1c75fb/python/taichi/lang/__init__.py#L293-L312? Thanks!

Reply:
Sure thing.